src/toggleButton/index.tsx

```diff
@@ -1,14 +1,16 @@
 import React, { PureComponent } from 'react'
 import cc from 'classcat'
 
-import Item, { ItemProps } from '_utils/item/index'
+import Item from '_utils/item/index'
+import { ItemProps } from '_utils/item/Item'
```

Comment:
nit: you also export it on the index

Reply:
It's not for use outside of the library, so we should keep it internally. Also I ran into a weird issue with typescript when trying to import/export interfaces 🤔